### PR TITLE
NETOBSERV-268 frontend code styling

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

- [X] Sort imports
- [X] Constants naming
- [X] Usage of <Flex> <FlexItem> vs css display: flex
- [X] Usage of <Text>
- [X] Define interface for props every time
- [X] Move non component files to utils (filters-helper.ts, metrics-helper.tsx)
- [ ] Split netflow-traffic code that became too big for a single file => will be done in a followup
- [X] Split other files (panels / contents)

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [X] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

**Review by jotak — APPROVED** [review]
nice!
/lgtm

(edit: ah that's still a draft ... but feel free to merge and open another one if this can avoid you troubles rebasing if kept open for too long .. as you want)

**Comment by jpinsonneau** [comment]
> nice! /lgtm
> 
> (edit: ah that's still a draft ... but feel free to merge and open another one if this can avoid you troubles rebasing if kept open for too long .. as you want)

Sorry @jotak I had other changes in my list
I think we should merge that first and I'll work on the `netflow-traffic` page split right after

**Review by stleerh — COMMENTED** [review]
Just one minor comment.  Everything else /lgtm.

**Comment by jotak** [comment]
@jpinsonneau LGTM, just leaving you checking the failing cypress test and Steven's comment

File: web/src/components/netflow-traffic.css [hunk 1]
```diff
@@ -235,4 +229,20 @@ span.pf-c-button__icon.pf-m-start {
...
+.netobserv-tooltip-text {
+    color: #fff;
+}
+
+.netobserv-no-child-margin>* {
+    margin: 0 !important;
+    margin-top: 0 !important;
+    margin-right: 0 !important;
+    margin-bottom: 0 !important;
+    margin-left: 0 !important;
```
[the remark this answers is not in the document]
Comment (jpinsonneau) [reply web/src/components/netflow-traffic.css]: sure, addressed in https://github.com/netobserv/network-observability-console-plugin/pull/541/commits/1120718388878fcdd6d573d48c55e9c1032ede5f

**Comment by jpinsonneau** [comment]
> @jpinsonneau LGTM, just leaving you checking the failing cypress test and Steven's comment

not sure what was causing the issue. Let's see if it happens again

**Comment by jpinsonneau** [comment]
@memodi any idea on what could cause the cypress error ? 

```
  netflow-topology
    ✓ displays topology and namespaces (14093ms)
    1) find network observability namespace
Warning: vkCreateInstance: Found no drivers!
Warning: vkCreateInstance failed with VK_ERROR_INCOMPATIBLE_DRIVER
    at CheckVkSuccessImpl (../../third_party/dawn/src/dawn/native/vulkan/VulkanError.cpp:88)
    at CreateVkInstance (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:458)
    at Initialize (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:344)
    at Create (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:266)
    at operator() (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:521)
    ✓ update options (62983ms)
  2 passing (2m)
  1 failing
  1) netflow-topology
       find network observability namespace:
     AssertionError: Timed out retrying after 10000ms: Expected to find element: `.pf-c-chart`, but never found it.
      at Context.eval (webpack://netobserv-plugin/./cypress/e2e/topology/topology.spec.ts:41:26)
```

**Comment by jpinsonneau** [comment]
Seems like https://github.com/netobserv/network-observability-console-plugin/pull/541/commits/1caa117807843e2bda5170ab3d19ba5ce6d6280f fixed it :smile_cat:

**Comment by memodi** [comment]
> Seems like [1caa117](https://github.com/netobserv/network-observability-console-plugin/commit/1caa117807843e2bda5170ab3d19ba5ce6d6280f) fixed it 😸

glad, you were able to figure this out. I'd not have idea. Perhaps we should enable video recording and archiving images from failed runs to the build logs?

**Comment by memodi** [comment]
cc @Amoghrd this has been merged, were there any regression test impact?

**Comment by Amoghrd** [comment]
Havent tested it. Will run regression. Was waiting for a reply on JIRA

**Comment by Amoghrd** [comment]
Regression is passing [cypress-tests#664](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/cypress-tests/664/console) with some flakes. All passed on manual rerun. But table_queryopts and netflow_table tests required changes; Made here: [openshift-tests-private#17547](https://github.com/openshift/openshift-tests-private/pull/17547)

**Pull request merged**